src/card.css

```diff
@@ -0,0 +1,114 @@
+.card {
+  background: var(--clear_white-color);
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card--position {
+  display: flex;
+  flex-direction: column;
+  padding: 0 16px;
+  max-width: 200px;
+}
+
+.card__logo {
+  align-self: center;
+  height: 134px;
+  width: 160px;
+  margin: 32px 0 40px;
+}
+
+.card__description {
+  font-family: Roboto, Arial, Helvetica, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  margin-bottom: 4px;
+  color: var(--dark_blue-for-text);
+}
+
+.card__code {
+  margin-bottom: 16px ;
+  color: var(--gray-for-text);
+}
+
+.card__raiting {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 24px;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(./images/star.svg);
+  background-position: 50%;
+  background-repeat: no-repeat;
+  height: 16px;
+  width: 16px;
+}
+
+.stars__star:not(:last-child) {
+  margin-right: 4px;
+}
+
+.stars__star:nth-child(n):not(.stars__star:nth-child(5)) {
+  background-image: url(./images/star-active.svg);
+}
```

Comment:
the checklist, point 11

Reply:
[FILE STRUCTURE] - You should create separate file for each BEM block
Fix it pls